web/src/enterprise/campaigns/detail/AddChangesetForm.tsx

```diff
@@ -90,31 +90,48 @@ export const AddChangesetForm: React.FunctionComponent<{ campaignID: ID; onAdd:
     )
     return (
         <>
-            <Form className="form-inline" onSubmit={submit}>
-                <input
-                    required={true}
-                    type="text"
-                    size={35}
-                    className="form-control mr-1"
-                    placeholder="Repository name"
-                    value={repoName}
-                    onChange={event => setRepoName(event.target.value)}
-                />
-                <input
-                    required={true}
-                    type="text"
-                    size={16}
-                    className="form-control mr-1"
-                    placeholder="Changeset number"
-                    value={externalID}
-                    onChange={event => setExternalID(event.target.value)}
-                />
+            <h3>Track changeset</h3>
+            <Form onSubmit={submit}>
+                <div className="d-flex">
+                    <div className="form-group mr-3 mb-0">
+                        <label htmlFor="changeset-repo">Repository name</label>
+                        <input
+                            required={true}
+                            name="changeset-repo"
```

Comment:
```suggestion
                            id="changeset-repo"
```

Same suggestion as below

Reply:
👍